R/modules_grammar_hpc.R

```diff
@@ -861,78 +966,78 @@ get_single_cell.HPCell = function(input_hpc, factors_to_regress = NULL, target_i
 #' @param .contrasts Contrasts parameter.
 #' @return The result of the differential abundance test.
 #'
-setMethod(
```

Comment:
There is no need to comment this out, it is just a method and you can choose to not use it. Again, if you commented you're creating a lot of noise for that things you have changed a lot of stuff, making it a bit difficult for code reviewers.